WordPress/Classes/ViewRelated/Plans/PlanComparisonViewController.swift

```diff
@@ -0,0 +1,177 @@
+    }
+    
+    @IBAction func pageControlChanged() {
+        guard let index = currentIndex else { return }
+        
+        if pageControl.currentPage > index {
+            scrollToNextPage() { success in
+                if success {
+                    self.currentPlan = self.allPlans[index + 1]
+                }
+            }
+        } else if pageControl.currentPage < index {
+            scrollToPreviousPage() { success in
+                if success {
+                    self.currentPlan = self.allPlans[index - 1]
+                }
+            }
+        }
+    }
+
+    // MARK: - PageViewController
+    private func configurePageViewController() {
+        pageViewController.dataSource = self
+        pageViewController.delegate = self
+        
+        if let index = currentIndex,
+            let initialViewController = viewControllerAtIndex(index) {
+                
+            pageViewController.setViewControllers([initialViewController], direction: .Forward, animated: false, completion: nil)
+        }
+    }
+    
+    private func viewControllerAtIndex(index: Int) -> UIViewController? {
+        guard index >= 0 && index < allPlans.count  else {
+            return nil
+        }
+        
+        let plan = allPlans[index]
+        return PlanDetailViewController.controllerWithPlan(plan)
+    }
+    
+    private func indexOfViewController(viewController: PlanDetailViewController) -> Int {
+        let plan = viewController.plan
+        return allPlans.indexOf(plan)!
+    }
+    
+    private func updatePageControl() {
+        if let index = currentIndex {
+            pageControl?.currentPage = index
+        }
+    }
+}
+
+extension PlanComparisonViewController: UIPageViewControllerDataSource, UIPageViewControllerDelegate {
+    func pageViewController(pageViewController: UIPageViewController, viewControllerBeforeViewController viewController: UIViewController) -> UIViewController? {
+        return previousViewController()
+    }
+    
+    func pageViewController(pageViewController: UIPageViewController, viewControllerAfterViewController viewController: UIViewController) -> UIViewController? {
+        return nextViewController()
+    }
+    
+    private func previousViewController() -> UIViewController? {
+        guard let index = currentIndex else { return nil }
+        
+        return viewControllerAtIndex(index - 1)
+    }
+    
+    private func nextViewController() -> UIViewController? {
+        guard let index = currentIndex else { return nil }
+        
+        return viewControllerAtIndex(index + 1)
+    }
+    
+    func pageViewController(pageViewController: UIPageViewController, willTransitionToViewControllers pendingViewControllers: [UIViewController]) {
+        if let controller = pendingViewControllers.first as? PlanDetailViewController {
+            currentPlan = allPlans[indexOfViewController(controller)]
+        }
+    }
```

Comment:
I think we should drop this one. It felt a bit odd that the title and page control updated as soon as you start dragging. If you look at the iOS home screen, the page control only updates when the transition finishes.